<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StageFilterGUI</class>
 <widget class="QWidget" name="StageFilterGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Selected processing stage:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="playbackGroupBox">
     <property name="title">
      <string>Playback</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="preMixCheckBox">
        <property name="text">
         <string>Pre-mix</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="postMixCheckBox">
        <property name="text">
         <string>Post-mix</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="captureGroupBox">
     <property name="title">
      <string>Capture</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="captureCheckBox">
        <property name="text">
         <string>Capture</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
